models/_sector/nft/trades/solana/platforms/magiceden_v2_solana_trades.sql

```diff
@@ -0,0 +1,243 @@
+{{
+    config(
+        schema = 'magiceden_v2_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        ,materialized = 'incremental'
+        ,file_format = 'delta'
+        ,incremental_strategy = 'merge'
+        ,incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+        ,unique_key = ['project','trade_category','outer_instruction_index','inner_instruction_index','account_mint','tx_id']
+        ,post_hook='{{ expose_spells(\'["solana"]\',
```

Comment:
when refactoring, we may want to remove the post-hook to expose project-level, as the intention is to push users to the final downstream spell `nft_solana.trades` to query

Reply:
this applies to all project spells, will only comment here